### PR TITLE
Refine calc distance function

### PR DESCRIPTION
**Pull request by yhmo**
Signed-off-by: groot <yihua.mo@zilliz.com>

#17102

### CODE REVIEW

**Comment by yhmo** [comment]
rerun ut

**Comment by xiaofan-luan** [comment]
@yhmo do we need review on the issue?

File: internal/proxy/impl.go [hunk 1]
```diff
@@ -3080,15 +3068,20 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
 			queryShardPolicy:   roundRobinPolicy,
 		}
 
+		items := make([]zapcore.Field, 0)
+		items = append(items, zap.String("traceID", traceID))
+		items = append(items, zap.String("role", typeutil.ProxyRole))
+		items = append(items, zap.Int64("msgID", qt.Base.MsgID))
+		items = append(items, zap.Uint64("timestamp", qt.Base.Timestamp))
```
Comment (soothing-rain) [inline internal/proxy/impl.go:3075]: I don't think we want all of them logged, for example we definitely don't need "traceID", "role", "msgID" and "db"

File: internal/proxy/impl.go [hunk 2]
```diff
@@ -3080,15 +3068,20 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
...
 		err := node.sched.dqQueue.Enqueue(qt)
 		if err != nil {
-			log.Debug("CalcDistance queryTask failed to enqueue",
-				zap.Error(err),
-				zap.String("traceID", traceID),
-				zap.String("role", typeutil.ProxyRole),
-				zap.String("db", queryRequest.DbName),
-				zap.String("collection", queryRequest.CollectionName),
-				zap.Any("partitions", queryRequest.PartitionNames))
+			items = append(items, zap.Error(err))
```
Comment (soothing-rain) [inline internal/proxy/impl.go:3083]: Can just put in a single line:

log.Error("CalcDistance queryTask failed to enqueue", append(items, zap.Error(err))...)

File: internal/proxy/impl.go [hunk 3]
```diff
@@ -3080,15 +3068,20 @@ func (node *Proxy) CalcDistance(ctx context.Context, request *milvuspb.CalcDista
 			queryShardPolicy:   roundRobinPolicy,
 		}
 
+		items := make([]zapcore.Field, 0)
+		items = append(items, zap.String("traceID", traceID))
```
Comment (soothing-rain) [inline internal/proxy/impl.go:3072]: items := []zapcore.Field {
	zap.String("traceID", traceID),
	zap.String("role", typeutil.ProxyRole),
	zap.Int64("msgID", qt.Base.MsgID),
	...
}

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**